ocrd_tool.md

```diff
@@ -19,6 +19,27 @@ Optionally, workflow processors can be notified that this file is potentially
 large and static (e.g. a fixed dataset or a precomputed model) and should be cached
 indefinitely after download by setting the `cacheable` property to `true`.
 
+The filename itself, i.e. the concrete value `<fpath>` of a file parameter
+should be resolved in the following way:
+
+* If `<fpath>` is an `http`/`https` URL: Download to a temporary directory (if
+  `cacheable==False`) or a semi-temporary cache directory (if `cacheable==True`)
+* If `<fpath>` is an absolute path: Use as-is.
+* If `<fpath>` is a relative path, try resolving the following paths and return
+  the first one found if any, raise a `FileNotFoundError` otherwise:
+  * `$CWD/<fpath>`
```

Comment:
I would welcome this from a user perspective, but it might be harder to implement (because we loose CWD context as soon as we `chdir` to the workspace after processor invocation).

Reply:
That is indeed a bit tricky in OCR-D/core but that should not prevent us. It's a common sense convention and we will find a way to implement it in core.